<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrategySettingClass</class>
 <widget class="QDialog" name="StrategySettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>跟单策略设置</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>价格策略</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>开仓单策略</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>委托        秒不成交撤单</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>撤单后以             价格偏离        跳追单</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>追单        次后不成交</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="openActionSec">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>21</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="openPriceType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>51</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>委托价格</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最新价格</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="openJump">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>51</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="openTimes">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>81</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="openFinal">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>81</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>撤单</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>挂单</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QComboBox" name="followType">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>样本成交价跟单</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>市价跟单</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>291</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>平仓单策略</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>委托        秒不成交撤单</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>撤单后以             价格偏离        跳追单</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>追单        次后不成交</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="closeActionSec">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>21</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="closePriceType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>51</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>委托价格</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最新价格</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="closeJump">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>51</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="closeTimes">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>81</y>
      <width>43</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="closeFinal">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>81</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>撤单</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>挂单</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>followType</tabstop>
  <tabstop>openActionSec</tabstop>
  <tabstop>openPriceType</tabstop>
  <tabstop>openJump</tabstop>
  <tabstop>openTimes</tabstop>
  <tabstop>openFinal</tabstop>
  <tabstop>closeActionSec</tabstop>
  <tabstop>closePriceType</tabstop>
  <tabstop>closeJump</tabstop>
  <tabstop>closeTimes</tabstop>
  <tabstop>closeFinal</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>StrategySettingClass</receiver>
   <slot>saveBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveBtnClicked()</slot>
 </slots>
</ui>
